src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.behaviour.FigureCreation;
+import core.basesyntax.behaviour.FigureDraw;
+import core.basesyntax.tools.Color;
+import java.util.Random;
+
+public abstract class Figure implements FigureDraw, FigureCreation {
+    protected static final int MAX_SIDE_FIGURE = 14;
+    private Color colorFigure;
+    private String name;
+    private double area;
+
+    protected static int randomSideFigure() {
+        return new Random().nextInt(MAX_SIDE_FIGURE) + 1;
+    }
+
+    public double getArea() {
+        return area;
+    }
+
+    public void setArea(double area) {
+
+        this.area = area;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getColor() {

```

Comment:
This method should return color, in our program color is enum, so please return propriate value here